API/fleece/Fleece.h

```diff
@@ -718,6 +722,10 @@ while (NULL != (value = FLDictIterator_GetValue(&iter))) {
         Its initial ref-count is 1, so a call to FLMutableDict_Free will free it.  */
     FLMutableDict FLMutableDict_New(void) FLAPI;
 
+    /** Creates a new mutable Dict from json. The input JSON must represent a JSON array, or NULL will be returned.
+        Its initial ref-count is 1, so a call to FLMutableDict_Free will free it.  */
```

Comment:
FLMutableDict_Release instead of FLMutableDict_Free?

Reply:
yes. copy/paste error. Fixed